scripts/build/plugins/uglify.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+const { minify } = require("uglify-es");
```

Comment:
Doesn't `rollup-plugin-uglify-es` do this?

Reply:
I don't know.. we were already using it like this, just moved to a separate file

Reply:
Actually original rollup plugin uglify uses uglify es.

Reply:
To be clear, I just copied what we were doing already in the old `scripts/build/rollup.parser.config.js`